webrender/src/tiling.rs

```diff
@@ -960,11 +959,13 @@ pub struct RenderPass {
 }
 
 impl RenderPass {
-    fn new(pass_index: isize, is_framebuffer: bool) -> RenderPass {
+    fn new(pass_index: isize,
+           is_framebuffer: bool,
+           size: DeviceUintSize) -> RenderPass {
```

Comment:
let's store the size inside a `RenderPass` in order to avoid asking for it in `allocate_target`?